libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -3945,7 +3945,7 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                             }
                             else
                             {
-                                error = $"{expr} can't evaluate.";
+                                error = $"{expr} should contain a string of URI.";
```

Comment:
>string of URI [](start = 66, length = 13)

URI string

Reply:
changed to  should contain a URI string.
Also applied to the below ones